lib/storage/data/external/AwsClient.js

```diff
@@ -317,6 +317,8 @@ class AwsClient {
     uploadPart(request, streamingV4Params, stream, size, key, uploadId,
         partNumber, bucketName, log, callback) {
         let hashedStream = stream;
+        let callbackCalled = false;
+
```

Comment:
I think we have a utility for that, `jsutil.once`

Reply:
done here : https://github.com/scality/Arsenal/pull/2254/files#diff-d75a9a19ab15f67df9624710374912016460445162dff59507eba0a27dbbdec2R8